rosdep/python.yaml

```diff
@@ -6229,6 +6229,11 @@ python3-jsonschema:
   fedora: [python3-jsonschema]
   gentoo: [dev-python/jsonschema]
   ubuntu: [python3-jsonschema]
+python3-junitparser:
+  debian: [python3-junitparser]
+  fedora: [python3-junitparser]
+  gentoo: [dev-python/junitparser]
```

Comment:
@danthony06 is it or is it not available? I can't seem to find these.

Reply:
It is unavailable. I will remove them.